\chapter{Building a theoretical model for the dynamics and self-organization of centromeres}

\section{Introduction}

The centromere identity is epigenetically determined by the histone H3 variant CENP-A in organisms with regional centromeres \citep{Warburton1997ImmunolocalizationCentromeres, Vafa1997ChromatinPlate, Earnshaw1985ThreeChromosome, Liu2006MappingCells, Regnier2005CENP-ABubR1, Heun2006, Mendiburo2011, Barnhart2011, Logsdon2015}. Contrary to the epigenetic nature, the chromosomal position of a centromere is stably inherited over generations of cells and only changes if viewed from an evolutionary timescale \citep{Amor2004HumanProgress, Murphy2005DynamicsMaps}. This implies the existence of strategies ensuring the propagation, maintenance and specificity of CENP-A at centromeres. Indeed, various delicate molecular mechanisms conserved across species have been found. 

The replication of DNA during the S phase inevitably dilutes the number of CENP-A nucleosomes in one chromosome at least by half, where full conservation of old CENP-A is assumed. Therefore, the propagation of CENP-A nucleosomes over generations relies on the deposition of new CENP-A into the chromatin. A conserved positive feedback mechanism has emerged across different species, where old CENP-A is read by proteins that can recruit the CENP-A specific chaperon to deposit new CENP-A \citep{Stirpe2022, McKinley2015}. In vertebrates, CENP-A binds the CCAN protein CENP-C and together they recruit the Mis18 complex, consisting of Mis18$\alpha$, Mis18$\beta$ and Mis18BP1 \citep{Westhorpe2015AMaintenance, Moree2011CENP-CAssembly, Dambacher2012CENP-CChromatin, French2017XenopusAssembly, Wang2014MitoticHJURP}. The CENP-A chaperon HJURP bearing CENP-A-H4 heterodimer is then targeted to the centromere by the Mis18 complex \citep{Foltz2009, Dunleavy2009}. Similarly, in fission yeast, its Mis18 complex, composed of Mis16, Mis18 and Eic1, and the CENP-A chaperon Scm3 are required for the assembly of its CENP-A ortholog Cnp1 at the centromere \citep{Pidoux2009FissionChromatin, Hayashi2004Mis16Centromeres, Williams2009FissionChromatin}, albeit the interaction with Cnp3, the fission yeast CENP-C, is not conserved \citep{Subramanian2014Eic1Assembly}. Mis18 complex and HJURP orthologs are not found in \textit{Drosophila}. But their functions are combined in one single protein CAL1, which not only possesses the CENP-A chaperon activity but also can self-target to the centromere by interacting with CENP-C and \textit{Drosophila} CENP-A CID \citep{Chen2014, MedinaPritchard2020, Phansalkar2012EvolutionaryDrosophila, Roure2019, Schittenhelm2010}. Additional factors are required for CID deposition in this system, including the FACT complex \citep{Chen2015EstablishmentTranscription} and CAF1 \citep{Furuyama2006Chaperone-mediatedVitro, Boltengagen2016AMelanogaster}.

Similar to other epigenetic marks, the maintenance of CENP-A nucleosomes is challenged by chromatin remodelling activities such as replication and transcription. Yet, CENP-A possesses unusual stability that it does not turn over once incorporated into the chromatin \citep{Falk2015, Jansen2007, Bodor2013, Smoak2016Long-TermIdentity}. In replication, this has been attributed to 


% self-propagation over generations
%   - observation
%   - the positive feedback
%   - CENP-B
%   - transcription
% stably incorporated 
%   - observation
%   - transcription
%   - replication
% centromere enrichment but low density
%   - the quantitative study
%   - regulation by PLK1 and CDK
%   - ectopic site eviction (replication, expression timing, PTM)
% island pattern
%   - observation 

\nomenclature{CENP-A}{CENtromere Protein A}
\nomenclature{CCAN}{Constitutive Centromere Associated Network}
\nomenclature{HJURP}{Holliday JUnction Recognition Protein}
\nomenclature{CID}{Centromere IDentifier}
 
\section{Methods}
\section{Results}
\section{Discussion}
