\chapter{Building a theoretical model for the dynamics and self-organization of centromeres}

\section{Introduction}

The centromere identity is epigenetically determined by the histone H3 variant CENP-A in organisms with regional centromeres \citep{Warburton1997ImmunolocalizationCentromeres, Vafa1997ChromatinPlate, Earnshaw1985ThreeChromosome, Liu2006MappingCells, Regnier2005CENP-ABubR1, Heun2006, Mendiburo2011, Barnhart2011, Logsdon2015}. Contrary to the epigenetic nature, the chromosomal position of a centromere is inherited over generations of cells with astonishing fidelity and only changes if viewed from an evolutionary timescale \citep{Amor2004HumanProgress, Murphy2005DynamicsMaps}. This implies the existence of strategies ensuring the propagation, maintenance and specificity of CENP-A at centromeres. Indeed, various delicate molecular mechanisms conserved across species have been found. 

DNA replication during the S phase inevitably dilutes the number of CENP-A nucleosomes in one chromosome at least by half, where full conservation of old CENP-A is assumed. Therefore, the propagation of CENP-A nucleosomes over generations relies on the deposition of new CENP-A into the chromatin. A conserved positive feedback mechanism has emerged across different species, where old CENP-A is read by proteins that can recruit the CENP-A specific chaperon to deposit new CENP-A \citep{Stirpe2022, McKinley2015}. In vertebrates, CENP-A binds the CCAN protein CENP-C and together they recruit the Mis18 complex, consisting of Mis18$\alpha$, Mis18$\beta$ and Mis18BP1 \citep{Westhorpe2015AMaintenance, Moree2011CENP-CAssembly, Dambacher2012CENP-CChromatin, French2017XenopusAssembly, Wang2014MitoticHJURP, Pan2019MechanismLicensing}. The CENP-A chaperon HJURP bearing CENP-A-H4 heterodimer is then targeted to the centromere by the Mis18 complex \citep{Foltz2009, Dunleavy2009}. CENP-B, the centromeric protein that binds the 17-bp DNA motif CENP-B box, is reported to reinforce this process by stabilising CENP-C \citep{Fachinetti2015, Hoffmann2020, Chardon2022CENP-B-mediatedCentromeres, Masumoto1989ASatellite., Suzuki2004CENP-BLocalization} and is believed to be the 'safety net' for the positive feedback loop \citep{Berg2020}. Similarly, in fission yeast, its Mis18 complex, composed of Mis16, Mis18 and Eic1, and the CENP-A chaperon Scm3 are required for the assembly of its CENP-A ortholog Cnp1 at the centromere \citep{Pidoux2009FissionChromatin, Hayashi2004Mis16Centromeres, Williams2009FissionChromatin}, albeit the requirement for Cnp3, the fission yeast CENP-C, is not conserved \citep{Subramanian2014Eic1Assembly}. Mis18 complex and HJURP orthologs are not found in \textit{Drosophila}. But their functions are combined in one single protein CAL1, which possesses the CENP-A chaperon activity and can self-target to the centromere by interacting with CENP-C and \textit{Drosophila} CENP-A CID \citep{Chen2014, MedinaPritchard2020, Phansalkar2012EvolutionaryDrosophila, Roure2019, Schittenhelm2010}. Additional factors are required for CID deposition in this system, including the FACT complex \citep{Chen2015EstablishmentTranscription} and CAF1 \citep{Furuyama2006Chaperone-mediatedVitro, Boltengagen2016AMelanogaster}. Even in point centromere species budding yeast, whose centromeres are determined genetically, this mechanism is partially conserved. The CBF3 complex recognises the centromeric sequence CDEIII and recruits the chaperon Scm3 to deposit the CENP-A ortholog Cse4 \citep{Camahort2007Scm3Kinetochore, Guan2021StructuralFormation, Cho2011Ndc10Yeast, Mizuguchi2007NonhistoneNucleosomes, Zhou2011StructuralScm3, Meluh1998Cse4pCerevisiae}. Although the exact timing might range from late M to early G1 phase, a common feature of CENP-A deposition shared by various species is that it happens outside the S phase, unlike canonical histone proteins \citep{Fukagawa2014, McKinley2015TheFunction, Stirpe2022}. Consistent with this feature, in vertebrates, the centromeric localisation of HJURP and Mis18BP1 is prevented by CDK1/2-dependent phosphorylation, whose activity is elevated in mitosis and reduced in interphase \citep{Silva2012, Spiller2017MolecularDeposition, Stankovic2017}. The deposition machinery is further regulated by PLK1, which phosphorylates Mis18BP1 and promotes its localisation to the centromere \citep{McKinley2014Polo-likeCentromeres}. Apart from the positive feedback loop and phospho-regulation, proper deposition of CENP-A requires centromeric transcription \citep{Bergmann2012EpigeneticFunction, Catania2015SequenceChromatin, Cardinale2009HierarchicalModifier, Choi2011IdentificationCentromeres, Nakano2008InactivationModifiers, Zhu2018HistoneChromosomes}. It is reasoned that transcription facilitates new CENP-A deposition by evicting embedded non-CENP-A nucleosomes \citep{Chen2015EstablishmentTranscription, Choi2011IdentificationCentromeres, Bobkov2018, Bergmann2012EpigeneticFunction, Bobkov2020, Choi2017TheH3, Prasad2011NewCentromeres}. 

Similar to other epigenetic marks, the maintenance of CENP-A nucleosomes is challenged by chromatin remodelling activities such as replication and transcription. Yet, CENP-A possesses unusual stability that it does not turn over once incorporated into the chromatin \citep{Falk2015, Jansen2007, Bodor2013, Smoak2016Long-TermIdentity}. In replication, this has been attributed to the recycling of disrupted CENP-A-H4 heterodimer by HJURP interacting with MCM2 of the replication fork \citep{Zasadzinska2018, Zasadzinska2013DimerizationDeposition, Huang2015AForks} in a similar manner to the retention of canonical H3 by Asf1$\alpha$ \citep{Richet2015StructuralFork, Clement2015MCM2Fork}. This resulted in the conservative partition of existing CENP-A nucleosomes between the newly replicated sister chromatids \citep{Falk2015, Jansen2007, Bodor2013}. Due to the temporal separation of replicative dilution and new CENP-A deposition, the gaps generated are then filled by another histone H3 variant H3.3 as the placeholder \citep{Dunleavy2011H3.3Phase.}. As mentioned above, transcription can lead to the eviction of incorporated nucleosomes. The general chaperon FACT complex and Spt6 have been proposed to travel with RNA Pol II and reassemble CENP-A nucleosomes dissembled due to transcription \citep{Bobkov2020, Jeronimo2019HistoneModifications, Kato2013Spt6H3, Boltengagen2016AMelanogaster}.

Counter-intuitively, the majority of CENP-A nucleosomes are localised on chromosome arms \citep{Bodor2014}. This could be due to the random deposition of CENP-A caused by the large quantity and the promiscuity to general histone chaperons. Nevertheless, the centromeric enrichment is still over 50-fold higher than arms \citep{Bodor2014}. Apart from the isolation of CENP-A deposition from canonical histone proteins mentioned above, nucleosome eviction by replication, gene expression regulation and PTMs are used to improve the specificity of CENP-A at the centromere. Replication is the main mechanism to remove ectopic CENP-A nucleosomes \citep{Nechemia-Arbely2019, Wang2021PhosphorylationCycle}, with the centromeric ones being proposed to be protected by the CCAN \citep{Nechemia-Arbely2019}. In human and fission yeast cells, CENP-A transcription coincides with their respective deposition timings \citep{Shelby1997AssemblySites, Takahashi2000RequirementYeast, Aristizabal-Corrales2019CellFormation}. Moreover, it has been observed that the expression of CENP-A outside the normal time window would lead to ectopic incorporation in different organisms \citep{Aristizabal-Corrales2019CellFormation, Heun2006, Tomonaga2005CentromereAneuploidy, Au2008AlteredCerevisiae, Olszak2011, McGovern2012CentromereCancer, Athwal2015CENP-ACells, Shrestha2021, Moreno-Moreno2019TheCycle}. Among the various  PTM-based mechanisms for regulating CENP-A localisation, ubiquitin-mediated protein degradation is the most common one \citep{Stirpe2022}. In human \citep{Maehara2010CENP-AMitoses, Lomonte2001DegradationICP0}, \textit{Drosophila} \citep{Moreno-Moreno2019TheCycle, Bade2014TheManner} and budding yeast \citep{Ranjitkar2010AnDomain, Au2013AProteolysis, Mishra2015Pat1Ubiquitination, Zhou2021MolecularPsh1} cells, ubiquitination has been reported to remove CENP-A outside the centromeric regions. 

Despite that extensive studies have been conducted on the molecular biology of centromere specification and propagation, our understanding of the system is compromised due to the absence of a mechanistic mathematical model. The different enrichment of CENP-A nucleosomes at centromeres and arms implicated the existence of two stable equilibrium states, or bi-stability, in the system. Positive feedback is known for its capability to generate bi-stability in biological systems \citep{Mitrophanov2008PositiveSystems, Ferrell2013FeedbackCycle}. It is tempting to hypothesise that the molecular mechanism of CENP-A deposition is sufficient to explain the difference in enrichment between centromeres and arms. Yet, the signal amplifier, or 'all or nothing', nature of positive feedback loops contrasts the low density of CENP-A nucleosomes at centromeres, where they only account for about 1 in 25 nucleosomes \citep{Bodor2014, Schittenhelm2010}. CENP-A further possesses intriguing features in terms of spatial localisation on the chromatin. Rather than distributed homogeneously, the CENP-A nucleosomes are visualised as distinct clusters on an extended chromatin fibre \citep{Blower2002ConservedHumans, Dunleavy2011H3.3Phase., Kyriacou2018}. It would be interesting to understand the mechanism behind this pattern formation. Therefore, this project aims to build a theoretical model describing the dynamics and spatial patterns of CENP-A, with the expectation to recapitulate the key characteristics of the system, including bi-stability, low density at the higher steady state and the maintenance of island patterns. 

\nomenclature{CENP-A}{CENtromere Protein A}
\nomenclature{CCAN}{Constitutive Centromere Associated Network}
\nomenclature{HJURP}{Holliday JUnction Recognition Protein}
\nomenclature{CID}{Centromere IDentifier}
\nomenclature{FACT}{Facilitates Chromatin TranscripTion}
 
\section{Methods}
\subsection{The model}

Inspired by the classic theoretical model for epigenetics \citep{Dodd2007, Micheelsen2010TheoryLandscapes}, we decided to use a CA-like, 1D, rule-based stochastic model for the CENP-A system. As a starting point, we developed a basic model, where the molecular knowledge of CENP-A propagation and maintenance is described in its simplest manner. The assumptions used for the basic model are as follows: 

(1) A certain number of sequentially placed nucleosomes composed of either canonical H3 or CENP-A nucleosomes was considered to represent the centromere. Because the lengths of centromeres vary largely among species and even between different chromosomes in the same species, we did not set a fixed value for the number of nucleosomes. As will be shown in the later section, this variable only provides a minor effect on the model's behaviours. Periodic boundary conditions were used to avoid potential artefacts from boundaries. 

(2) At each time step, CENP-A nucleosomes are first replenished and then diluted (Figure~\ref{fig:basicmodelschematics}B). This is because CENP-A is deposited at the interphase while diluted at the S phase in most species. The two processes will be referred to as 'replenishment' and 'dilution' in the following texts for convenience. We assume no loss of CENP-A except replicative dilution as the ultra-low turnover rate of CENP-A nucleosomes mentioned above. 

(3) For replenishment, H3 nucleosomes were converted to CENP-A nucleosomes because of pre-existing CNEP-A nucleosomes in proximity (Figure~\ref{fig:basicmodelschematics}A). Notably, the positive feedback of CENP-A deposition was simplified as its local self-promoting property in this case. The detailed algorithm of this process will be described in the following implementation sub-section. 

(4) For dilution, CENP-A nucleosomes were assumed to be randomly distributed to daughter centromeres as observed in wet-lab experiments (Figure~\ref{fig:basicmodelschematics}A). 

\begin{figure}[htbp]
  \centering
  \includegraphics[width=0.9\textwidth]{chapter2/figures/the model.pdf}
  \caption[Schematics of the basic model]{Schematics of the basic model. (A) Schematics of the interchange between CENP-A and H3 nucleosomes. CENP-A nucleosomes are converted to H3 nucleosomes due to replicative dilution. H3 nucleosomes are converted to CENP-A nucleosomes by replenishment, where pre-existing CENP-A nucleosome facilitates the conversion of H3 nucleosomes locally. (B) An example of the evolution of the basic model. The array of nucleosomes undergoes replenishment and dilution at each cell cycle. }
  \label{fig:basicmodelschematics}
\end{figure}

\nomenclature{CA}{Cellular Automata}
\nomenclature{1D}{one-Dimensional}

\subsection{Implementation}

The model describes the centromere as a 1D array of a certain number, denoted by $NN$, of cells with two states, either 1, representing the CENP-A nucleosome or 0, representing the H3 nucleosome.

\subsubsection{Initialisation}

To unbiasedly initialize such an array, a stochastic approach is used. a 1D array of 0 was first created. Each 0 in the array then has a probability of the arbitrary initial CENP-A density, denoted by $\rho_{0}$, to be converted to 1. A typical array was exemplified in Figure~\ref{fig:array}. The density and spatial pattern of CENP-A are used as read-outs of the state of the  array. Density is calculated by dividing the sum of the array, which equals the number of 1s in the array, by the length of the array. As a mimicry of CENP-A imaging data from biological experiments, spatial pattern is visualized by plotting the array with 1 as white and 0 as black. \\

\begin{figure}[htbp]
  \centering
  \includegraphics[width=0.9\textwidth]{chapter2/figures/the array.pdf}
  \caption[The example array]{The example array. The centromere is described as a finite 1D array composed of only 0 and 1, with 0 representing the H3 nucleosome and 1 representing the CENP-A nucleosome. The density of CENP-A is calculated by dividing the sum of the array by the length, or the number of cells, of the array. The spatial pattern of CENP-A distribution is visualised by colouring 1 as white and 0 as black. }
  \label{fig:array}
\end{figure}

\subsubsection{Replenishment}

As with other CA models, our model updates the state of a cell by a state-transition function that reads the states of neighbour cells. To describe the replenishment, where new CENP-A nucleosomes replace H3 nucleosomes due to pre-existing CENP-A and the CENP-A nucleosomes do not turn over, we set the state-transition function as such: 
\begin{align*}
    & P_{0\rightarrow 1}=\alpha\sum_{i=-n}^{n} x_if_i \\
    & P_{1\rightarrow 0}=0 
\end{align*}

        
where $\alpha$ is an arbitrary parameter denoting the loading efficiency, $n$ denotes the size of the neighbourhood, $x$ is the state of the cell $i$ and $f$ is the weight function, which is a Gaussian function with a mean of 0 and a standard deviation of n/3 ($f\sim Gaussian(\mu=0, \sigma=\frac{n}{3}$) used to model the assumption that the probability of interaction decreases with physical distance. 

To enable more flexibility of replenishment, we allowed it to happen multiple rounds before dilution. The number of rounds was denoted by $rr$. 

\subsubsection{Dilution}

Ideally, one array should be divided into two daughter arrays during each dilution event. Yet, the number of arrays would grow exponentially with the simulation steps, as would the computational time required. To simulate the evolution of the array for more steps, only one of the daughter arrays was tracked. Hence, replicative dilution is modelled by the process where each 1 in the array has a probability of 0.5 to be converted to 0, which leads to the following state-transition function: 
\begin{align*}
    & P_{0\rightarrow 1}=0 \\
    & P_{1\rightarrow 0}=0.5
\end{align*}

The parameters of the basic model were summarised in Table~\ref{tab:parameters}. \\

\begin{table}[htbp]
\centering
\caption{The parameters of the basic model}
\label{tab:parameters}
\begin{tabular}{cl}
\hline
\textbf{Parameters} & \multicolumn{1}{c}{\textbf{Description}} \\ \hline
$NN$                  & The number of nucleosomes                \\
$\rho_{0}$               & The initial density of CENP-A            \\
$\alpha$               & The  loading efficiency                  \\
$n$                   & The size of neighbourhood                \\
$rr$                  & The number of replenishment rounds      
\end{tabular}
\end{table}

\section{Results}
\subsection{The basic model exhibits two types of behaviour}

We started by experimenting with different combinations of parameters and classifying behaviours of the basic model based on CENP-A density evolution maps (Figure~\ref{fig:modelBehaviour}A and B left). A density evolution map presents how CENP-A density changes over time. In general, two types of behaviour were observed. First, the density eventually
reaches 0 (Figure~\ref{fig:modelBehaviour}A). Due to the fact that new CENP-A nucleosomes require old CENP-A nucleosomes to be recruited, once the density reaches 0, it will stay at 0. We named this type of behaviour ‘dead’. The second type is where CENP-A density is fluctuating around a value less than 0.5 after a number of generations (Figure~\ref{fig:modelBehaviour}B). We called this type of behaviour ‘stabilized’. The maximal value of density cannot be greater than 0.5 because we update the array by first replenishing and then diluting it. Then we sought to visualise the evolution of the CENP-A spatial pattern of the two behaviours by stacking spatial patterns according to generation. We termed this plot spatial pattern kymograph (Figure~\ref{fig:modelBehaviour}A and B right). For the ‘dead’ type behaviour, CENP-A nucleosomes formed inheritable aggregates, or islands, which could diverge or be combined. But all the islands vanished eventually. For the ‘stabilized’ behaviour, the islands could be seen in the first several time steps. After that, the islands joined each other and formed random-like distribution of CENP-A. Given the auto-amplification property of CENP-A deposition, the existence of ‘stabilized’ behaviours, where CENP-A density can be stabilised before saturation, is surprising. We reasoned that this is because as the density of CENP-A nucleosome increases, the number of H3 nucleosomes decreases, leading to a reduced room for new CENP-A to be deposited. This counteracts auto-amplification and results in a steady state. 

We next attempted to systematically view how model behaviours change with parameters. Due to the predictable positive effect on the CENP-A density of $rr$, I fixed it and ran simulations with different combinations of the loading efficiency $\alpha$ ranging from 0 to 1 with a step size of 0.02 and the range of local deposition $\sigma$ ranging from 0 to 10 with a step size of 0.2. Each simulation was conducted for 500 time steps and assigned to either ‘dead’ or ‘stabilised’ type based on whether the average density of the first 100 time steps is less or greater than the last 100 time steps. The result showed that the two behaviours fall into two separate phases (Figure~\ref{fig:modelBehaviour}C). Both $\alpha$ and $\sigma$ positively correlated to the ‘stabilized’ behaviour. Interestingly, the boundary between the two phases can roughly be described by a reciprocal function.


\begin{figure}[htbp]
  \centering
  \includegraphics[width=0.9\textwidth]{chapter2/figures/model_behaviour.pdf}
  \caption[Typical behaviours of the basic model.]{Typical behaviours of the basic model. The simulations were conducted at $NN$=1000, $\rho_{0}$=0.02, $\sigma$=3, $rr$=3 if not otherwise stated. (A) The density evolution map and spatial pattern kymographs of a representative ‘dead’ behaviour ($\alpha$=0.45). (B) The density evolution map and spatial pattern kymographs of a representative ‘stabilised’ behaviour ($\alpha$=0.7). (C) Phase diagram of the model behaviour as a function of $\alpha$ and $\sigma$. Black represents the ‘dead’ behaviour and white represents the ‘stabilised’ behaviour for a specific parameter set. (D) Bifurcation diagram of asymptotic density as a function of $\alpha$. 10 lineages were run for 10,000 (blue) or 100,000 time steps (orange) and the average density of the last 2,000 time steps was plotted as the asymptotic density. }
  \label{fig:modelBehaviour}
\end{figure}

\begin{figure}[htbp]
  \centering
  \includegraphics[width=0.9\textwidth]{chapter2/figures/parameter_test.pdf}
  \caption[The model behaviours are insensitive to stochasticity, nucleosome number and initial density.]{The model behaviours are insensitive to stochasticity, nucleosome number and initial density. The simulations were conducted at $NN$=500, $\rho_{0}$=0.02, $\alpha$=0.7, $\sigma$=3, $rr$=3 if not otherwise stated. (A) The density evolution maps of different simulations of the representative ‘dead’ or 'stabilised' behaviour. 10 simulations were conducted for each behaviour, shown in different colours. (B) The density evolution map of the representative ‘stabilised' behaviour with different $NN$s. (C) The density evolution map of the representative ‘stabilised' behaviour with different $\rho_{0}$s. For each $\rho_{0}$, 100 simulations were conducted for 30 time steps. Each colour represents the average of 100 simulations. }
  \label{fig:parameterTest}
\end{figure}

\subsection{Mean-field approximation is able to qualitatively explains the simulation results}
\subsection{Global cooperative deposition generates bi-stability but cannot maintain the higher steady state at a low density}
\subsection{Arbitrary number control leads to the formation of few CENP-A clusters}
\subsection{Spontaneous conversion can recapitulate the island pattern but not bi-stability}
\section{Discussion}
